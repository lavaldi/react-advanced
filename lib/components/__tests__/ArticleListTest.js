import React from 'react';
import ArticleList from '../ArticleList';

// import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('ArticleList', () => {
  const testProps = {
    articles: {
      a: { id: 'a' },
      b: { id: 'b' }
    }
  };

  it('renders conrrectly', () => {
    const wrapper = shallow(
      <ArticleList
        {...testProps}
      />
    );
    console.log(wrapper.getElement());

    // expect(tree.children.length).toBe(2);
    // expect(tree).toMatchSnapshot();
  });
});
