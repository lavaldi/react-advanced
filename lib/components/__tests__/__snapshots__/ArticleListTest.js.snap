// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList renders conrrectly 1`] = `
<div>
  <div>
    <div />
    <div>
      Invalid Date
    </div>
    <div>
      <a
        href={undefined}
      >
         
      </a>
    </div>
    <div />
  </div>
  <div>
    <div />
    <div>
      Invalid Date
    </div>
    <div>
      <a
        href={undefined}
      >
         
      </a>
    </div>
    <div />
  </div>
</div>
`;
